lib/auth/grpcserver.go

```diff
@@ -618,6 +618,11 @@ func (g *GRPCServer) GenerateUserCerts(ctx context.Context, req *authpb.UserCert
 	}
 
 	if req.Purpose == authpb.UserCertsRequest_CERT_PURPOSE_SINGLE_USE_CERTS {
+		if req.MFAResponse != nil {
+			if _, ok := req.MFAResponse.Response.(*authpb.MFAAuthenticateResponse_TOTP); ok {
+				return nil, trace.BadParameter("per-session MFA is not satisfied by 'otp' devices, please update your configuration to use WebAuthn")
```

Comment:
What does "your configuration" refer to in this context? Is there something more specific we could replace it with?

Reply:
We also shouldn't generally make it to this error so I think it's probably fine just to state what went wrong :+1: